File: app/src/main/java/ru/michanic/mymot/UI/Adapters/ExpandableListAdapter.java
package ru.michanic.mymot.UI.Adapters;

import android.content.Context;
import android.util.DisplayMetrics;
import android.widget.ExpandableListView;

public class ExpandableListAdapter extends ExpandableListView {

    public ExpandableListAdapter(Context context)
    {
        super(context);
    }

    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
    {
        DisplayMetrics lDisplayMetrics = getResources().getDisplayMetrics();
        widthMeasureSpec = MeasureSpec.makeMeasureSpec(lDisplayMetrics.widthPixels, MeasureSpec.AT_MOST);
        heightMeasureSpec = MeasureSpec.makeMeasureSpec(1000000, MeasureSpec.AT_MOST);
        super.onMeasure(widthMeasureSpec, heightMeasureSpec);
    }

}

File: app/src/main/java/ru/michanic/mymot/Kotlin/UI/Adapters/ExpandableListAdapter.kt
package ru.michanic.mymot.Kotlin.UI.Adapters

import android.content.Context
import android.widget.ExpandableListView

class ExpandableListAdapter(context: Context?) : ExpandableListView(context) {
    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        var widthMeasureSpec = widthMeasureSpec
        var heightMeasureSpec = heightMeasureSpec
        val lDisplayMetrics = resources.displayMetrics
        widthMeasureSpec =
            MeasureSpec.makeMeasureSpec(lDisplayMetrics.widthPixels, MeasureSpec.AT_MOST)
        heightMeasureSpec = MeasureSpec.makeMeasureSpec(1000000, MeasureSpec.AT_MOST)
        super.onMeasure(widthMeasureSpec, heightMeasureSpec)
    }
}